docs/content/direct/best-practices.md

```diff
@@ -0,0 +1,9 @@
+# Best Practices
+
+**Note:**  This section is under constration and includes partial information. 
+## Size considerations
+As KubeStellar is built on top of Kubernetes all Kuberentes limitations and recommendations apply to KubeStellar as well. These recommendations can be found in [Kubernetes Considerations for large clusters](https://kubernetes.io/docs/setup/best-practices/cluster-large/).
+
+In addition, KubeStellar Transport Plugin is built on top of OCM, so KubeStellar also comply to some of OCM's limitations. Users should take into account the following restrictions:
```

Comment:
Link to OCM's limitations?

Reply:
Didn't find one :(

Reply:
I couldn't find the limits documentation either, but note that in the ocm code, the limit is 500K not 512K https://github.com/open-cluster-management-io/ocm/blob/ff77abd31208aaf328b1dc0826b66caf9cdf6dcd/pkg/work/webhook/common/validator.go